kie-soup-project-datamodel/kie-soup-project-datamodel-commons/src/main/java/org/kie/soup/project/datamodel/commons/util/RawMVELEvaluator.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.soup.project.datamodel.commons.util;
+
+import java.util.Map;
+
+import org.mvel2.MVEL;
+import org.mvel2.compiler.CompiledExpression;
+import org.mvel2.integration.VariableResolverFactory;
+
+public class RawMVELEvaluator implements MVELEvaluator {
```

Comment:
@mbarkley Is this meant to be the default implementation?

I'm unable to run ```drools-wb-webapp``` as there are no CDI Beans satisfying injection points for ```MVELEvaluator``` in ```org.kie.workbench.common.services.backend.enums.EnumDropdownServiceImpl```. I can make this implementation ```@ApplicationScoped``` and add a dependency for ```javax.enterprise:cdi-api``` however IDK if ```kie-soup``` is meant to be CDI-free and we need a _producer_ method in kie-wb-common/guvnor instead!?

Reply:
OK... I needed ```drools-cdi``` dependency too. All fixed.